packages/cli/templates/default-app/packages/backend/package.json.hbs

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "backend",
+  "version": "0.0.0",
+  "main": "dist/index.cjs.js",
+  "types": "src/index.ts",
+  "private": true,
+  "engines": {
+    "node": "12"
+  },
+  "scripts": {
+    "build": "backstage-cli backend:build",
+    "build-image": "backstage-cli backend:build-image example-backend",
+    "start": "backstage-cli backend:dev",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "clean": "backstage-cli clean",
+    "migrate:create": "knex migrate:make -x ts"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^{{version}}",
+    "@backstage/catalog-model": "^{{version}}",
+    "@backstage/config": "^0.1.1-alpha.13",
+    "@backstage/config-loader": "^0.1.1-alpha.13",
```

Comment:
@Rugvip do you know why this works only for this particular version and not if I use `"^{{version}}"` here?

Reply:
Yep, needs https://github.com/spotify/backstage/pull/1734